gradle/libs.versions.toml

```diff
@@ -48,6 +49,8 @@ jetbrains-annotations = { module = "org.jetbrains:annotations", version.ref = "j
 aws-lambda-events = { module = "com.amazonaws:aws-lambda-java-events", version.ref = "aws-lambda-events" }
 aws-lambda-serialization = { module = "com.amazonaws:aws-lambda-java-serialization", version.ref = "aws-lambda-serialization" }
 graal-svm = { module = "org.graalvm.sdk:nativeimage", version.ref = "graal-svm" }
+lombok = { module = "org.projectlombok:lombok", version.ref = "lombok" }
```

Comment:
really want to open that can of worms? Maybe just write the builder manually?

Reply:
not really but I don't know another way to test